recipes/pyrobex/LICENSE

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
This is already included in the PyPI tarball so you shouldn't include it manually in the recipe.